strongbox-commons/pom.xml

```diff
@@ -206,7 +206,15 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
         </dependency>
+
         <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
+            <version>3.11.4</version>
```

Comment:
This version change will actually have to be done in the [strongbox-parent](https://github.com/strongbox/strongbox-parent/) instead. That's the only place we define versions. These dependencies are then inherited from there.